File: src/kyu_6/two_sum/js/solution.js
function twoSum(numbers, target) {
    for (let i = 0; i < numbers.length; i++) {
        for (let j = 0; j < numbers.length; j++) {
            if (i !== j) {
                if (numbers[i] + numbers[j] === target) {
                    return [i, j];
                }
            }
        }
    }
    return [0, 1];
}

module.exports = twoSum;

File: src/kyu_6/two_sum/cpp/solution.cpp
std::pair<std::size_t, std::size_t> two_sum(const std::vector<int>& numbers, int target) {
    for(auto i = 0; i < numbers.size(); i++)
      for(auto j = i+1; j < numbers.size(); j++)
        if(numbers[i] + numbers[j] == target) return {i, j};
}